x/posts/internal/types/msgs_test.go

```diff
@@ -75,7 +75,7 @@ func TestMsgCreatePost_ValidateBasic(t *testing.T) {
 			error: sdkerrors.Wrap(sdkerrors.ErrInvalidAddress, "Invalid creator address: "),
 		},
 		{
-			name: "Empty message returns error",
+			name: "Empty message returns error if medias are empty",
```

Comment:
I'm not a big fan of editing already existing test cases in this way. We should instead add newer ones that test with: 

- empty message, empty medias
- empty message, non empty medias
- non empty message, empty medias
- non empty message, non empty medias